Makefile

```diff
@@ -71,3 +72,23 @@ board_server_status:
 	systemctl status board_server
 
 .PHONY: setup sim-test fpga-test doc-test test-all board_server_install board_server_uninstall board_server_status
+
```

Comment:
This Makefile does not need to be changed as we will not use it for now

Reply:
It is used only to call targets in the build directory, which can be called there before things are stable.